is/is.go

```diff
@@ -25,36 +32,33 @@ func ValueAny[T any](v T) bool {
 	return reflectValue(&v)
 }
 
-func reflectValue(vp any) bool {
-	switch rv := reflect.ValueOf(vp).Elem(); rv.Kind() {
-	case reflect.Map, reflect.Slice:
-		return rv.Len() != 0
-	default:
-		return !rv.IsZero()
-	}
-}
-
-// ValueSlice returns true if the length of the slice is greater than 0.
-// Note that it does not distinguish nil slices from empty slices.
-func ValueSlice[T any, S ~[]T](v S) bool {
-	return len(v) > 0
-}
-
-// ValueMap returns true if the length of the map is greater than 0.
+// TruthyMap returns true if the length of the map is greater than 0.
 // Note that it does not distinguish nil maps from empty maps.
-func ValueMap[K comparable, V any, M ~map[K]V](v M) bool {
+func TruthyMap[K comparable, V any, M ~map[K]V](v M) bool {
 	return len(v) > 0
 }
 
-// Value returns the truthy value of comparable types.
+// Get returns the truthy value of dereferenced pointers of comparable types.
 // Values are truthy if they are not equal to the zero value for the type.
-func Value[T comparable](v T) bool {
-	return v != *new(T)
+// Note that it will evaluate to true for double pointers.
+func TruthyPointer[T comparable](v *T) bool {
+	if v == nil {
+		return false
+	}
+	return Truthy(*v)
+}
+
+// TruthySlice returns true if the length of the slice is greater than 0.
+// Note that it does not distinguish nil slices from empty slices.
+func TruthySlice[T any, S ~[]T](v S) bool {
+	return len(v) > 0
```

Comment:
`is.TruthySlice(s)` is longer than `len(s) > 0`. Is the clarity worth it? Maybe if it were a little shorter?

Reply:
what do you think about [e66db9d](https://github.com/carlmjohnson/truthy/pull/2/commits/e66db9d72ad3de562654c0c4a46df1ed265f469b)?